seismic/pick_harvester/quality.py

```diff
@@ -58,8 +58,8 @@ def function(x, A, B):
         ps = np.fabs(cwt) ** 2
         ps = pywt.threshold(ps, np.std(ps), mode='soft', substitute=1)
 
-        psbefore   = ps[:, :ps.shape[1] / 2]
```

Comment:
these two lines look a bit cryptic. It would be good to add inline #comments explaining what they try to achieve.